src/bika/lims/content/abstractbaseanalysis.py

```diff
@@ -547,9 +547,9 @@
                      'ResultText': _('Display Value'), },
     subfield_validators={'ResultValue': 'result_options_value_validator',
                          'ResultText': 'result_options_text_validator'},
-    subfield_sizes={'ResultValue': 5,
+    subfield_sizes={'ResultValue': 20,
                     'ResultText': 25,},
-    subfield_maxlength={'ResultValue': 5,
+    subfield_maxlength={'ResultValue': 40,
                         'ResultText': 255,},
```

Comment:
I agree that 5 might be too small, but at probably saves some horizontal screen space when an analysis has many interims defined as well.

Off topic: It would be great to have some JS that allows fields to grow according to their value

Reply:
Indeed @ramonski,  and looks like the underlying html/js-template consumes these values under the hood which could be modified to do as you suggest.